src/current/_includes/releases/v24.3/v24.3.0-beta.2.md

```diff
@@ -0,0 +1,33 @@
+## v24.3.0-beta.2
+
+Release Date: October 28, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v24-3-0-beta-2-sql-language-changes">SQL language changes</h3>
+
+- If a table is the destination of a logical data replication stream, then only schema change statements that are deemed safe are allowed on the table. Safe statements are those that do not result in a rebuild of the primary [index]({% link v24.3/indexes.md %}) and do not create an index on a virtual [computed column]({% link v24.3/computed-columns.md %}). [#133266][#133266]
```

Comment:
This and a couple other nodes in Operational Changes reference "replication stream" and I wonder if there is a good link.

Should LDR be in Enterprise Edition Changes?

Reply:
There aren't any docs yet for LDR. I will come along and link as I'm working on them. Technically I suppose this could be in Enterprise Edition... but does that matter with the Core deprec coming up?

Reply:
I will leave the notes where they are given the Core deprecation.